libraries/AP_Param/AP_Param.cpp

```diff
@@ -1789,9 +1803,20 @@ bool AP_Param::load_defaults_file(const char *filename)
         return false;
     }
 
-    if (! read_param_defaults_file(filename)) {
-        return false;
+    saveptr = nullptr;
+    mutable_filename = strdup(filename);
+    if (mutable_filename == nullptr) {
+        AP_HAL::panic("AP_Param: Failed to allocate mutable string");
+    }
+    for (char *pname = strtok_r(mutable_filename, ",", &saveptr);
+         pname != nullptr;
+         pname = strtok_r(nullptr, ",", &saveptr)) {
+        if (!read_param_defaults_file(pname)) {
+            free(mutable_filename);
+            return false;
+        }
```

Comment:
Well... I actually started writing something along those lines and then thought, "wait, isn't there a library function for just this sort of thing?" :-)

Performance *really*, *really* isn't our concern at this point in the code :-)

I see a lot of pointer manipulation for no huge benefit here.  Perhaps if we did have a strv of some description it would be less intimidating in that respect.

Still, I'm not really fussed either way.  The functionality is good.